plugins/blip/basics/logic/datamunger.js

```diff
@@ -188,24 +188,23 @@ module.exports = function(bgClasses) {
 
       return null;
     },
-    processInfusionSiteHistory: function(basicsData, latestPump, patient) {
+    processInfusionSiteHistory: function(basicsData, latestPump, patient, permissions) {
       if (!latestPump) {
         return null;
       }
 
       var {
-        permissions,
         profile: {
           fullName,
         },
         settings,
       } = patient;
 
-      var hasUploadPermission = permissions.hasOwnProperty('admin') || permissions.hasOwnProperty('custodian') || permissions.hasOwnProperty('root');
+      var canUpdateSettings = permissions.hasOwnProperty('admin') || permissions.hasOwnProperty('custodian') || permissions.hasOwnProperty('root');
```

Comment:
Just a note to make sure we don't forget: as Darin confirmed, we can drop `admin`.